portal-frontend/src/app/features/edit-submission/proposal/pfrs-proposal/pfrs-proposal.component.ts

```diff
@@ -120,19 +116,11 @@ export class PfrsProposalComponent extends FilesStepComponent implements OnInit,
           maximumDepth: applicationSubmission.soilToPlaceMaximumDepth ?? undefined,
         };
 
-        let isNOIFollowUp = null;
-        if (applicationSubmission.soilIsNOIFollowUp !== null) {
-          isNOIFollowUp = applicationSubmission.soilIsNOIFollowUp ? 'true' : 'false';
-          if (isNOIFollowUp) {
-            this.NOIIDs.enable();
-          }
-        }
-
-        let hasALCAuthorization = null;
-        if (applicationSubmission.soilHasPreviousALCAuthorization !== null) {
-          hasALCAuthorization = applicationSubmission.soilHasPreviousALCAuthorization ? 'true' : 'false';
-          if (hasALCAuthorization) {
-            this.applicationIDs.enable();
+        let isFollowUp = null;
+        if (applicationSubmission.soilIsFollowUp !== null) {
+          isFollowUp = applicationSubmission.soilIsFollowUp ? 'true' : 'false';
```

Comment:
just nitpick, there is parseBooleanToString that could handle this

Reply:
Done! Cleaned up all 3 soil components which was quite a reduction. good idea